core/src/test/java/io/questdb/test/griffin/GroupByTest.java

```diff
@@ -1675,6 +1675,567 @@ public void testSelectMatchingButInDifferentOrderThanGroupBy() throws Exception
+                            "FROM hits\n" +
+                            "WHERE CounterID = 62 AND EventTime >= '2013-07-01T00:00:00Z' AND EventTime <= '2013-07-31T23:59:59Z' AND IsRefresh = 0\n" +
+                            "GROUP BY TraficSourceID, SearchEngineID, AdvEngineID, Src, Dst\n" +
+                            "ORDER BY PageViews DESC\n" +
+                            "LIMIT 1000, 1010;";
+            assertPlan(query1, "Sort light lo: 1000 hi: 1010\n" +
+                    "  keys: [PageViews desc]\n" +
+                    "    VirtualRecord\n" +
+                    "      functions: [TraficSourceID,SearchEngineID,AdvEngineID,Src,Dst,PageViews]\n" +
+                    "        Async JIT Group By workers: 1\n" +
+                    "          keys: [TraficSourceID,SearchEngineID,AdvEngineID,Src,Dst]\n" +
+                    "          values: [count(*)]\n" +
+                    "          filter: (CounterID=62 and IsRefresh=0)\n" +
+                    "            DataFrame\n" +
+                    "                Row forward scan\n" +
+                    "                Interval forward scan on: hits\n" +
+                    "                  intervals: [(\"2013-07-01T00:00:00.000000Z\",\"2013-07-31T23:59:59.000000Z\")]\n");
+            String query2 =
+                    "SELECT TraficSourceID, SearchEngineID, AdvEngineID, CASE WHEN (SearchEngineID = 0 AND AdvEngineID = 0) THEN Referer ELSE '' END AS Src, URL, COUNT(*) AS PageViews\n" +
+                            "FROM hits\n" +
+                            "WHERE CounterID = 62 AND EventTime >= '2013-07-01T00:00:00Z' AND EventTime <= '2013-07-31T23:59:59Z' AND IsRefresh = 0\n" +
+                            "GROUP BY TraficSourceID, SearchEngineID, AdvEngineID, Src, URL\n" +
+                            "ORDER BY PageViews DESC\n" +
+                            "LIMIT 1000, 1010;";
+            assertPlan(query2, "Sort light lo: 1000 hi: 1010\n" +
+                    "  keys: [PageViews desc]\n" +
+                    "    VirtualRecord\n" +
+                    "      functions: [TraficSourceID,SearchEngineID,AdvEngineID,Src,URL,PageViews]\n" +
+                    "        Async JIT Group By workers: 1\n" +
+                    "          keys: [TraficSourceID,SearchEngineID,AdvEngineID,Src,URL]\n" +
+                    "          values: [count(*)]\n" +
+                    "          filter: (CounterID=62 and IsRefresh=0)\n" +
+                    "            DataFrame\n" +
+                    "                Row forward scan\n" +
+                    "                Interval forward scan on: hits\n" +
+                    "                  intervals: [(\"2013-07-01T00:00:00.000000Z\",\"2013-07-31T23:59:59.000000Z\")]\n");
+        });
+    }
+
+    @Test
+    public void testLiftAliasesFromInnerSelect7() throws Exception {
```

Comment:
Why don't we assert query plan here and below? Without such an assertion, it's hard to understand whether this new test is related to the changes.